logins-api/android/gradle.properties

```diff
@@ -11,3 +11,13 @@ org.gradle.jvmargs=-Xmx1536m
 # This option should only be used with decoupled projects. More details, visit
 # http://www.gradle.org/docs/current/userguide/multi_project_builds.html#sec:decoupled_projects
 # org.gradle.parallel=true
+
+libGroupId=org.mozilla.sync15
+libRepositoryName=application-services
+libProjectName=application-services
+libProjectDescription=Firefox Application Services
+libUrl=https://github.com/mozilla/application-services
+libVcsUrl=https://github.com/mozilla/application-services.git
+
+libLicense 'Apache-2.0'
```

Comment:
Probably should be MPL given the discussion the other day.

Reply:
Thanks for the reminder -- I will update, and perhaps I'll land #233 before this.